Makefile.am

```diff
@@ -38,6 +38,8 @@ AM_CPPFLAGS = \
 	-include $(top_builddir)/config.h \
 	-I$(top_builddir) \
 	-I$(top_srcdir) \
+	-I$(top_srcdir)/src/mavlink-router \
+	-I$(top_srcdir)/src/common \
```

Comment:
> One may suspect that `mainloop` and `pollable` classes belong to
> `src/common` - and they are right. However, currently, `mainloop` is the
> one class that perform the routing, so it will live on
> `src/mavlink-router` till it's converted to a pure mainloop.

I'd prefer to have this code move and the correspondent separation of mainloop from app in a separate PR (#82). And this PR to be done on top.  Code move is very easy to apply... this needs a thorough review.